spring-boot-project/spring-boot-docs/src/docs/asciidoc/web/spring-security.adoc

```diff
@@ -224,6 +224,37 @@ Again, the same properties are applicable for both servlet and reactive applicat
 Alternatively, you can define your own `OpaqueTokenIntrospector` bean for servlet applications or a `ReactiveOpaqueTokenIntrospector` for reactive applications.
 
 
+=== Audience Validation Support in OAuth2 Resource Server
```

Comment:
I don't think we need a new section to document the new configuration property and certainly not one at this level. I think we could just add to the Resource Server section above.